lib/util/esWrapper.js

```diff
@@ -154,21 +154,76 @@ class ESWrapper {
       }
     }
 
-    switch (error.meta.statusCode) {
-      case 400:
-        kuzzleError = kuzzleError || this._handleBadRequestError(error, message);
+    switch (error.displayName) {
+      case 'BadRequest':
+        if (kuzzleError) {
+          break;
+        }
+
+        if (error.body && error.body.error) {
+          message = error.body.error.root_cause
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Correct this attempt to modify "message" or use "let" in its declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3500)